src/common/include/log.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <spdlog/spdlog.h>
+
+class Log {
+ public:
+	static void Init();
+	[[nodiscard]] inline static std::shared_ptr<spdlog::logger> GetLogger() {
+		return logger_;
+	}
+
+ private:
+	static std::shared_ptr<spdlog::logger> logger_;
```

Comment:
You can use inline to avoid defenition in cpp https://www.codingame.com/playgrounds/2205/7-features-of-c17-that-will-simplify-your-code/inline-variables